frontend/src/scenes/max/components/FeedbackDisplay.tsx

```diff
@@ -0,0 +1,139 @@
+import './QuestionInput.scss'
+
+import clsx from 'clsx'
+import { useActions, useValues } from 'kea'
+import posthog from 'posthog-js'
+import { useCallback, useEffect } from 'react'
+
+import { LemonButton } from '@posthog/lemon-ui'
+
+import { FeedbackRating, FeedbackTriggerType } from '../FeedbackPrompt'
+import { feedbackPromptLogic } from '../feedbackPromptLogic'
+import { maxThreadLogic } from '../maxThreadLogic'
+
+export interface FeedbackDisplayProps {
+    isFloating?: boolean
+    conversationId: string
+}
+
+function captureMaxFeedback(
```

Comment:
This is the same thing as `captureNegativeFeedback` in `FeedbackPrompt` and in `feedbackPromptLogic` minus the feedback text, let's extract it and reuse it